ngraph/frontend/paddlepaddle/src/op/hard_swish.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "hard_swish.hpp"
+#include <ngraph/opsets/opset6.hpp>
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs hard_swish(const NodeContext& node)
+                {
+                    auto data = node.get_ng_input("X");
+                    if (node.has_attribute<float>("threshold"))
+                    {
+                        auto threshold = node.get_attribute<float>("threshold");
+                        PDPD_ASSERT(std::abs(threshold - 6.0) < 0.001,
+                                    "hard_swish: threshold must = 6.0.");
```

Comment:
I'd suggest to change assert message to 'Only threshold = 6.0 is currently supported'
And same for 'scale' and 'offset'